### ISSUE TITLE
Decide on JDE alternative

### ISSUE THREAD

**Issue by jakepeyser** [IBM-Bluemix/logistics-wizard#105]
**Ad-hoc**

As our access to JD Edwards has been postponed, we need to consider alternatives to using JDE as our back-end ERP system. The following options are on the table so far:
- [Odoo](https://www.odoo.com/)
- [SAP ERP](http://go.sap.com/product/enterprise-management/erp.html)

This issue will be used for our discussion around this selection process and feasibility of using either service.

**Comment by l2fprod** [comment]
some more alternatives:
https://en.wikipedia.org/wiki/List_of_ERP_software_packages#Free_and_open-source_ERP_software

**Comment by jakepeyser** [comment]
@shenybluemix @uwefassnacht: any response from your SAP contacts?

**Comment by uwefassnacht** [comment]
No, none yet. Will follow up with them.

**Comment by shenybluemix** [comment]
I have got some updates:
 
 - SAP S/4 HANA which is the SAP flagship ERP solution has the SaaS offering now.  There is an instance called "IDES" based SAP ERP 6.0 Enhancement Package 7 should have some data we need -> Not 100% sure since my experiences in SAP is very technical not business application focus
 
https://go.sap.com/cmp/oth/crm-s4hana/s4hana-on-premise.html
 
I have already register a S/4 HANA Trail account it is only for 14 days and it requires an AWS account.   
 
- SAP Business ByDesign is another SaaS offering targeting SME.  I am checking this in parallel 
Chris(Ye) Shen
Bluemix Developer Advocate/Offering Management
IBM Cloud  
 
Phone: 1-404-403-8362
E-mail: sheny@us.ibm.com
 
1001 E Hillsdale Blvd Suite 400Foster City, CA 94404-1603
United States
 
 
----- Original message -----From: Uwe Fassnacht <notifications@github.com>To: IBM-Bluemix/logistics-wizard <logistics-wizard@noreply.github.com>Cc: Ye Shen/Burlingame/IBM@IBMUS, Mention <mention@noreply.github.com>Subject: Re: [IBM-Bluemix/logistics-wizard] Decide on JDE alternative (#105)Date: Wed, Jun 29, 2016 5:48 AM 
No, none yet. Will follow up with them.

**Comment by jakepeyser** [comment]
ok, but remember we are not considering their SaaS offerings, we only want on-prem versions for our scenario

**Comment by jakepeyser** [comment]
We are engaged with Secure Gateway and SAP teams around using an existing SAP ERP instance on the SAP test network. [Details from last conversation](https://ibm.box.com/s/euukv092m1v2lyhp4eg0spqrpzvo5ci3)

**Comment by jakepeyser** [comment]
SAP access received and [connection details here](https://ibm.box.com/s/gvkbbf6231of7t00oco4w22eipmmqhix). Working on integration going forward
